app/views/layouts/_form_errors.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if defined? resource %>
```

Comment:
It looks like the builder has a [`govuk_error_summary` helper](https://github.com/DFE-Digital/govuk_design_system_formbuilder/blob/d12d51a74307f055df3a3b0ea36fbf1778366afa/lib/govuk_design_system_formbuilder/builder.rb#L430-L447) for this. But it expects to be inside the `<form>`...

Reply:
Talked in rl - not going to do this for the moment, as we're not sure we want it inside the form.